.gitlab/dogfood.yml

```diff
@@ -0,0 +1,33 @@
+dogfood-dogweb-failed:
+  stage: dogfood
+  tags: ["arch:amd64"]
+  needs: ["dogfood-dogweb-trigger"]
+  when: on_failure
+  script:
+    - exit 1
+
+dogfood-dogweb:
+  stage: dogfood
+  tags: ["arch:amd64"]
+  needs: ["dogfood-dogweb-trigger"]
+  when: on_success
+  script:
+    - exit 0
+
+dogfood-dogweb-trigger:
+  stage: dogfood
+  trigger:
+    project: DataDog/dogweb
+    strategy: depend
+    branch: emmett.butler/ddtrace-unstable-dogfooding
+  variables:
```

Comment:
Just an idea, not sure how easy/hard it is, but it would be cool if we could specify whether we wanted things like IAST to be enabled/disabled when we trigger this (or both by default).

Reply:
How have you been enabling it in recent tests?

Reply:
I had to do [these PRs](https://github.com/DataDog/dogweb/pull/119071), but I think you suggested that we could probably do something similar with the commit hash override for ddtrace, and also do an override of the IAST env var per trigger?

Reply:
Now that I mentioned it, I think _I_ was supposed to look into that 😅